packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageView.kt

```diff
@@ -234,5 +235,18 @@ class ExpoImageView(
       }
     }
   }
+
+  /**
+   * Called when Glide "injects" drawable into the view
+   */
+  override fun setImageDrawable(drawable: Drawable?) {
```

Comment:
Ass a comment here to explain why we're doing this 👀 
I know that it's pretty obvious after reading the method body, but let's have it in plain english here as well 🤞